docs/skopeo-copy.1.md

```diff
@@ -132,7 +132,7 @@ skopeo  copy --encryption-key jwe:./public.key --encrypt-layer 1 oci:local_nginx
 ```
 
 ## SEE ALSO
-skopeo(1), podman-login(1), docker-login(1)
+skopeo(1), podman-login(1), docker-login(1), containers-auth.json(5), containers\-signature(5), containers\-transports(5)
```

Comment:
is it time to change the podman-login refs to skopeo-login?

Reply:
or add skoepo-login(1) to it?

Reply:
Yes I removed all reference to Podman and replaced it with skopeo.